libs/community/langchain_community/callbacks/openai_info.py

```diff
@@ -25,15 +25,17 @@
     "gpt-4-vision-preview-completion": 0.03,
     "gpt-4-1106-preview-completion": 0.03,
     # GPT-3.5 input
-    "gpt-3.5-turbo": 0.0015,
+    "gpt-3.5-turbo": 0.0005,
```

Comment:
have base gpt-3.5-turbo prices changed? till 2/16 it still points at -0613 not -0125
![Screenshot 2024-02-05 at 6 11 14 PM](https://github.com/langchain-ai/langchain/assets/22008038/13777e08-321a-4685-a44f-fe5c2d27c722)

Reply:
yep, for 2 days gpt-3.5-turbo will be -0613 with price 0.0015

Reply:
but then it will be -0125 with new price

Reply:
ok! now its time!

Reply:
what makes you say that, isn't feb 16 the flip date?

Reply:
there is 
Customers using the pinned gpt-3.5-turbo model alias will be automatically upgraded from gpt-3.5-turbo-0613 to gpt-3.5-turbo-0125 two weeks after this model launches.